### PR TITLE
Add initial attack steps and associations for trainingLang

### LINKED ISSUE #31: Add models and resulting attack graphs created using a very simple trainingLang

### PR DESCRIPTION
**Pull request by mrkickling**

### CODE REVIEW

File: src/main/mal/trainingLang.mal [hunk 1]
```diff
@@ -20,22 +20,59 @@ category System {
...
+         fileSystemAccess
+
+    | connect
+      -> fileSystemAccess
+
+    | authenticate
+      -> fileSystemAccess,
+         networks.access
+
+    & fileSystemAccess
```
Comment (andrewbwm) [inline src/main/mal/trainingLang.mal]: I would just rename fileSystemAccess to access just to keep things simple and not make any inferences on how the access is achieved. It's not like we model the file system anyway.

File: src/main/mal/trainingLang.mal [hunk 2]
```diff
@@ -20,22 +20,59 @@ category System {
...
+
   }
 
 }
 
 associations {
-
+  Network [networks] * <-- NetworkHostAccess --> * [hosts] Host
+  User [usersOnHost] * <-- HostUsers --> * [hostsWithUser] Host
+  Data [dataOnHost] * <-- HostData --> 0..1 [hostWithData] Host
```
Comment (andrewbwm) [inline src/main/mal/trainingLang.mal]: I don't like the `host(s)WithX` nomenclature. We could either call them just `host(s)` or `containingHost` for Data or something along those lines.

**Comment by andrewbwm** [comment]
The category is called `System` which is a bit weird, but it contains `Host`, `Data`, `Network`, and `User`. I'm not sure it's the best name, but I don't really have a good suggestion for the name, it's also not really that important. But maybe we just call it `Training` or `Basics` if we want a generic name.

**Review by andrewbwm — APPROVED** [review]

File: src/main/mal/trainingLang.mal (hunk 2 above)
Comment (mrkickling) [reply src/main/mal/trainingLang.mal]: agreed!

Comment (mrkickling) [reply src/main/mal/trainingLang.mal]: Please see the latest updates

File: src/main/mal/trainingLang.mal (hunk 1 above)
Comment (mrkickling) [reply src/main/mal/trainingLang.mal]: Changed this in latest update

**Comment by mrkickling** [comment]
> The category is called `System` which is a bit weird, but it contains `Host`, `Data`, `Network`, and `User`. I'm not sure it's the best name, but I don't really have a good suggestion for the name, it's also not really that important. But maybe we just call it `Training` or `Basics` if we want a generic name.

changed System->Basics

**Pull request merged**